### PR TITLE
Implements most of Pipeline View

### PR DESCRIPTION
**Pull request by onyiny-ang**
Still need to figure out how to cut off the number of pipelineruns at 10. . .
Will attempt that next!

### CODE REVIEW

File: src/explorer/model.ts [hunk 1]
```diff
@@ -312,18 +337,40 @@ class PipelineNode implements TektonNode{
...
+  getPipelineTasks(): string[] {
+    return this.pipelineTasks;
+  }
+
+  parseToolTip(): string {
+    return this.tooltip.substring(this.tooltip.indexOf("-") +1);
+  }
+
+  parseEventName(event: any): string {
+    return event.object.metadata.name.substring(0, event.object.metadata.name.lastIndexOf("-run"))
```
Comment (vdemeester) [inline src/explorer/model.ts]: I am not sure about this test, does the event name always have `-run` in its name ?

**Comment by vdemeester** [comment]
Also, once the pipelinerun is done, it doesn't show anything anymore below the `cli-release-pipeline` (below)

File: src/explorer/model.ts (hunk 1 above)
Comment (onyiny-ang) [reply src/explorer/model.ts]: This is a question that I have too. This way of associating the pipeline with the pipeline run makes me very uneasy but I wasn't sure what would be a better way. Any suggestions?

Comment (vdemeester) [reply src/explorer/model.ts]: @onyiny-ang there should be a label `tekton.dev/Pipeline` on the `PipelineRun` with the reference name of the Pipeline

Comment (onyiny-ang) [reply src/explorer/model.ts]: Ok thanks. I will check that out and see what i can figure out :+1:

**Comment by onyiny-ang** [comment]
It seems that there is a race condition #7 that could be responsible for some of this. I'm working on trying to fix that now.

**Comment by onyiny-ang** [comment]
The most recent changes fix the issue with `-run` but I'm not entirely sure if the update problem is fix as I haven't been able to duplicate it. I suspect the second issue you found is related to the race condition and I'm still trying to figure out how best to handle that.
Another thing I'm not sure about is https://github.com/redhat-developer/vscode-tekton/blob/924d57a740d3a1816fde7f41e0ea0541462b1ee1/src/explorer/model.ts#L352
Right now the `toolTip` variable is the `[${event.object.metadata.namespace}]-${event.object.metadata.name}` and I'm not sure the function I have to parse it will necessarily always result in the pipeline name. . .

**Pull request merged**